processing/data_collection/gazette/spiders/es_vila_velha.py

```diff
@@ -0,0 +1,54 @@
+from datetime import date
+
+import dateparser
+from scrapy import FormRequest, Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class VilaVelhaSpider(BaseGazetteSpider):
+    name = "es_vila_velha"
+    allowed_domains = ["www.vilavelha.es.gov.br"]
+
+    TERRITORY_ID = "3205200"
+    GAZETTE_URL_CSS = "td:last-child a::attr(href)"
+    GAZETTE_DATE_CSS = "td:nth-child(2) span b::text"
+    GAZETTE_ISSUE_CSS = "td:nth-child(3) span b::text"
+    JAVASCRIPT_POSTBACK_REGEX = r"javascript:__doPostBack\('(.*)',''\)"
+
+    start_date = date(2016, 7, 1)
+
+    def start_requests(self):
+        date_param = self.start_date.strftime("%d/%m/%Y")
+        base_url = "https://www.vilavelha.es.gov.br/diariooficial/ConsultaDiario.aspx"
+        gazettes_url = f"{base_url}?dataInicial={date_param}"
+        yield Request(gazettes_url)
+
+    def parse(self, response):
+        table_head = "th"
+
+        for element in response.css("#ctl00_cpConteudo_gvDocumentos tr"):
+            if element.css(table_head).extract():
+                continue
```

Comment:
What do you think about this?
```suggestion
        for element in response.css("#ctl00_cpConteudo_gvDocumentos tr"):
            is_header = element.css("th").extract() != []
            if is_header:
            	continue
```

Reply:
Code review improvements

Reply:
I like it 👍